src/evilHangman/evilHangmanAsciiArt.ts

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs';
+
+const ART_PATH = './res/hangmen.txt';
+const FRAME_SEPARATOR = 'æ';
+const framesPromise = new Promise<Array<string>>((resolve, reject) => {
+	fs.readFile(ART_PATH, (err, data) => {
```

Comment:
Consider using the `readFile` variant from the `node:fs/promises` instead.

Reply:
oh nice, didn't know that existed.

Reply:
done